doc/index.rst

```diff
@@ -5,10 +5,9 @@
 
 .. container:: index-paragraph
 
-    Nilearn is a Python module for **fast and easy statistical learning on
-    NeuroImaging** data.
+    Nilearn is a Python module to enable **efficient statistical learning and analysis of brain volumes** through **approachable methods and tutorials**.
 
-    It leverages the `scikit-learn <http://scikit-learn.org>`__ Python toolbox
+    It supports GLM based analysis and leverages the `scikit-learn <http://scikit-learn.org>`__ Python toolbox
```

Comment:
Spelling out the acronym for naive readers :point_down: 

```suggestion
    It supports general linear model (GLM) based analysis and leverages the `scikit-learn <http://scikit-learn.org>`__ Python toolbox
```

Reply:
+1